Sources/Lumina/Camera/Extensions/Delegates/FileRecordingExtension.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  LuminaCamera+FileOutputRecordingDelegate.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+extension LuminaCamera: AVCaptureFileOutputRecordingDelegate {
+  func fileOutput(_ output: AVCaptureFileOutput, didFinishRecordingTo outputFileURL: URL, from connections: [AVCaptureConnection], error: Error?) {
+    DispatchQueue.main.async {
+      if error == nil, let delegate = self.delegate {
+        delegate.videoRecordingCaptured(camera: self, videoURL: outputFileURL)
+      }
+    }
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/Delegates/FileRecordingExtension.swift#L20"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>FileRecordingExtension.swift:20</code></td>
    </tr>
  </tbody>
</table>